apps/backend/src/bookings/bookings.repository.ts

```diff
@@ -5,16 +5,17 @@ import {
   FailedRelationConstraintError,
   RecordNotFound,
 } from './bookings.common';
+import { Guide } from 'types/src/dtos/guide/guide.dto';
 
 const bookingStatusEnumMapper = {
-  waitingForGuideConfirmation: BookingStatus.WAITING_FOR_GUIDE_CONFIRMATION,
-  guideCancelled: BookingStatus.GUIDE_CANCELLED,
-  waitingForPayment: BookingStatus.WAITING_FOR_PAYMENT,
-  waitingForRefund: BookingStatus.WAITING_FOR_REFUND,
-  waitingForTraveling: BookingStatus.WAITING_FOR_TRAVELING,
-  traveling: BookingStatus.TRAVELING,
-  finished: BookingStatus.FINISHED,
-  userCancelled: BookingStatus.USER_CANCELLED,
+  WAITING_FOR_GUIDE_CONFIRMATION: BookingStatus.WAITING_FOR_GUIDE_CONFIRMATION,
+  GUIDE_CANCELLED: BookingStatus.GUIDE_CANCELLED,
+  WAITING_FOR_PAYMENT: BookingStatus.WAITING_FOR_PAYMENT,
+  WAITING_FOR_REFUND: BookingStatus.WAITING_FOR_REFUND,
+  WAITING_FOR_TRAVELING: BookingStatus.WAITING_FOR_TRAVELING,
+  TRAVELING: BookingStatus.TRAVELING,
+  FINISHED: BookingStatus.FINISHED,
+  USER_CANCELLED: BookingStatus.USER_CANCELLED,
```

Comment:
don't sure way need to do like this 

may be use BookingStatus from db directly